src/lib.rs

```diff
@@ -171,8 +171,9 @@ pub use crate::postings::Postings;
 pub use crate::reader::LeasedItem;
 pub use crate::schema::{Document, Term};
 
-/// Expose the current version of tantivy, as well
-/// whether it was compiled with the simd compression.
+/// Expose the current version of tantivy as found in Cargo.toml during compilation.
+/// eg. "0.11.0"
+/// as well whether it was compiled with the simd compression.
```

Comment:
Does this method give any meaningful info about simd compression in tantivy?

Reply:
Ah no the SIMD copmression bit is irrelevant today. There used to be a different codec, but now tantivy uses a scalar fixture when the instruction set is not available. All platform use the same codec.